ext/patches/comet/CometSearch/CometWriteTxt.cpp

```diff
@@ -0,0 +1,701 @@
+/*

```

Comment:
Is this a new patch for Comet?

Reply:
Yes, this fixes a problem with two '-' being printed in the string representing mods when there are no mods, crux has been printing just one. We also make some changes to Common.h to enable compiling Comet on MacOS.